docs/config.md

```diff
@@ -660,27 +660,47 @@ export PROMREGATOR_AUTHENTICATOR_BASIC_PASSWORD=myPassword
 java -Dspring.config.location=file:/path/to/your/myconfig.yaml -jar promregator-0.0.1-SNAPSHOT.jar
 ```
 
-#### Option "promregator.authenticator.oauth2xsuaa.tokenServiceURL" (mandatory, if using promregator.authenticator.type=OAuth2XSUAA)
+#### Option "promregator.authenticator.oauth2xsuaa.tokenServiceURL" (mandatory, if using promregator.authenticator.type=OAuth2XSUAABasic)
 Specifies the URL of the OAuth2 endpoint, which contains the token service of your authorization server in case of global authentication. Typically, this is the endpoint with the path `/oauth/token`, as Promregator will try to perform to establish a ["Client Credentials"-based authentication](https://www.digitalocean.com/community/tutorials/an-introduction-to-oauth-2#grant-type-client-credentials).
 
-#### Option "promregator.authenticator.oauth2xsuaa.client_id" (mandatory, if using promregator.authenticator.type=OAuth2XSUAA)
+#### Option "promregator.authenticator.oauth2xsuaa.tokenServiceCertURL (mandatory for if using promregator.authenticator.type=OAuth2XSUAACertificate)"
+Specifies the URL of the OAuth2 endpoint for certificate based authentication, which contains the token service of your authorization server in case of global authentication.
+
+#### Option "promregator.authenticator.oauth2xsuaa.client_id" (mandatory, if using promregator.authenticator.type=OAuth2XSUAABasic or OAuth2XSUAACertificate)
 Specifies the client identifier (a.k.a. "client_id") which shall be used during the OAuth2 request based on the Grant Type Client Credentials flow in case of global authentication.
 
-#### Option "promregator.authenticator.oauth2xsuaa.client_secret" (mandatory, if using promregator.authenticator.type=OAuth2XSUAA)
-Specifies the client secret (a.k.a. "client_secret") which shall be used during the OAuth2 request based on the Grant Type Client Credentials flow in case of global authentication.
+#### Option "client_certificates" (mandatory if using promregator.authenticator.type=OAuth2XSUAACertificate)
+Specifies the certificate chain which shall be used during the OAuth2 request. Omitted when client secret-based authentification is used.
```

Comment:
The `omitted` part has been removed for `client_certificates´, `client_secret` and `client_key` since these parts are already implied by `mandatory if ...`.